# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1508632215 14400
#      Sat Oct 21 20:30:15 2017 -0400
# Node ID ef59a196c1557a48cd4cfcbbe8d4903119d86453
# Parent  44f0ad0e3af5d63d62f7f1933327411fbb6a2435
boostrap.conf: Set checkout_only_file to etc/HACKING.md.

diff --git a/bootstrap.conf b/bootstrap.conf
--- a/bootstrap.conf
+++ b/bootstrap.conf
@@ -145,17 +145,17 @@ fi
 
 # Build prerequisites
 buildreq="\
 autoconf   2.59
 automake   1.9.6
 git        1.5.5
 tar        -
 "
-checkout_only_file=etc/HACKING
+checkout_only_file=etc/HACKING.md
 gnulib_tool_option_extras="--libtool"
 gnulib_name="libgnu"
 source_base="libgnu"
 
 # Don't use git to update gnulib sources. We keep gnulib under a
 # Mercurial subrepository instead.
 use_git=false
 GNULIB_SRCDIR=gnulib
